third_party/ux-0.1.3/src/lib.rs

```diff
@@ -0,0 +1,997 @@
+//! # uX - non-standard-width integers types
+//!
+//! When non-standard-width integers is required in an applications, the norm is to use a larger container and make sure the value is within range after manipulation. uX aims to take care of this once and for all by:
+//!
+//! - Providing `u1`-`u127` and `i1`-`i127` types that should behave as similar as possible to the built in rust types
+//!     - The methods of the defined types are the same as for the built in types (far from all is implemented at this point but fill out an issue or create a PR if something essential for you is missing)
+//!     - Overflow will panic in debug and wrap in release.
+//! - All possible lossless conversions is possible by using `From`.
+//! - When `TryFrom` is stabilized fallible conversions will also be supported.
+
+
+#![cfg_attr(not(feature="std"), no_std)]
+
+
+mod lib {
+    pub mod core {
+        #[cfg(feature="std")]
+        pub use std::*;
+        #[cfg(not(feature="std"))]
+        pub use core::*;
+    }
+}
```

Comment:
I know you didn't write this code, but my opinion of lines 15 - 22 is "yuck".

Idk why Rust developers are so insistent on using the types in `std` whenever possible. If a crate optionally supports `no_std`, then it should just refer to things in `core` by their `core` path, not do this renaming.